src/main/java/br/com/caelum/vraptor/test/http/Parameters.java

```diff
@@ -38,9 +39,17 @@ public void fill(MockHttpServletRequest request) {
 		}
 		for (Parameter param : parameters) {
 			request.addParameter(param.getName(),param.getValue());
-		}		
+		}
 		for (Parameter param : headers) {
 			request.addHeader(param.getName(), param.getValue());
-		}		
+		}
+		if(bytes != null && bytes.length > 0){
+			request.setContent(bytes);
+		}
+	}
+
+	public Parameters setContent(byte[] bytes) {
```

Comment:
I think so, because org.springframework.mock.web.MockHttpServletRequest#content is byte[] type.
What type do you think? Is there another way?